client/packages/common/src/hooks/useNotification/useNotification.tsx

```diff
@@ -24,18 +29,47 @@ export const useNotification = (): NotificationHook => {
     </IconButton>
   );
 
+  const actionWithDetail = (key: SnackbarKey, message: string) => (
+    <>
+      <PaperClickPopover
+        placement="top"
+        width={300}
+        Content={
+          <PaperPopoverSection>
+            <Typography variant="body1">{message}</Typography>
+          </PaperPopoverSection>
+        }
+      >
+        <IconButton size="small">
+          <InfoIcon style={{ color: '#fff' }} />
+        </IconButton>
+      </PaperClickPopover>
+      {action(key)}
+    </>
+  );
+
   const error = (message: string, options?: OptionsObject) => () =>
     enqueueSnackbar(message, {
       variant: 'error',
       action,
       ...options,
     });
+
+  const errorWithDetail = (message: string, options?: OptionsObject) => () =>
+    enqueueSnackbar(`${t('label.error-info-icon')}:`, {
```

Comment:
Wondering if this might be good with a `error.something-wrong` first? 

 Then would read something like "Oops! Something's gone wrong. Click info icon to see error details: "